Lombiq.Analyzers/Build.props

```diff
@@ -63,7 +67,24 @@
     </ItemGroup>
   </Target>
 
-  <Target Name="Copy editorconfig" AfterTargets="AfterResolveReferences" Condition="'$(SolutionDir)' != '*Undefined*'">
-    <Copy SourceFiles="$(MSBuildThisFileDirectory).editorconfig" DestinationFolder="$(SolutionDir)" SkipUnchangedFiles="true" />
+  <PropertyGroup Condition="'$(NuGetBuild)' == 'true'">
+    <TreatWarningsAsErrors>True</TreatWarningsAsErrors>
+    <MSBuildTreatWarningsAsErrors>true</MSBuildTreatWarningsAsErrors>
+    <CodeAnalysisTreatWarningsAsErrors>True</CodeAnalysisTreatWarningsAsErrors>
+  </PropertyGroup>
+
+  <Target Name="CopyEditorconfig" DependsOnTargets="LombiqNetAnalyzers" AfterTargets="AfterResolveReferences">
+  </Target>
+
+  <!-- This target should be used in CI before build, e.g. `dotnet msbuild -target:LombiqNetAnalyzers | Out-Null`. Use
+       `| Out-Null` in PowerShell or `>/dev/null 2>&1` in bash to suppress output because it will complain if not every
+       project has this target, but that's not really relevant. If you are using NuGet, then the argument should be
+       "-target:Restore;LombiqNetAnalyzers" instead (include the quotes) to load the pacakge first.  -->
```

Comment:
That specific call is applicable for GHA, but I guess most Windows users don't have/know bash. Also it's not valid in Windows PowerShell either because the `||` operator is not supported there.

I replaced the "to load the pacakge first" with "to restore all package dependencies first".

Reply:
OK, ignore the `|| bash -c 'true'` part - I was trying to point to the fact that we use `-target:Restore;LombiqNetAnalyzers` ourselves, in contrast to the suggestion in the first line which is `-target:LombiqNetAnalyzers`. In that case the extra mention for the use of `nuget` (?) is just about the quotes `"`?

Reply:
It's for nuget.